Homework02.playground/Contents.swift

```diff
@@ -0,0 +1,60 @@
+struct Creature {
+    
+    var name: String
+    var description: String
+    var isGood: Bool
+    var magicPower: Int
+    
+    init(name: String, description: String, isGood: Bool, magicPower: Int) {
```

Comment:
Since all your properties are "built-in" Swift types, you do not need to define a constructor. Swift will do that for you.